.dbnd/airflow/unittests.cfg

```diff
@@ -0,0 +1,105 @@
+[core]
```

Comment:
Added by mistake?

Reply:
:0 yes, but I'm totally not sure how that happened. good catch thou

Reply:
I think this might be the result of playing around with kubernetes

Reply:
removed it